src/Backend/OnboardingInsuranceAPI/Extensions/StartupExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Cosmos;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using OnboardingInsuranceAPI.Services;
+
+namespace OnboardingInsuranceAPI.Extensions;
+
+internal static class StartupExtensions
+{
+    internal static IServiceCollection AddCosmosDb(this IServiceCollection services)
+    {
+        services.AddDbContext<DataContext>(optionsBuilder
+            => optionsBuilder.UseCosmos(
+            Environment.GetEnvironmentVariable("AccountEndpoint"),
+            Environment.GetEnvironmentVariable("AccountKey"),
+            databaseName: "Data",
+            options =>
+            {
+                options.ConnectionMode(ConnectionMode.Gateway);
+                options.WebProxy(new WebProxy());
+                options.Region(Regions.WestEurope);
+                options.GatewayModeMaxConnectionLimit(32);
+            }));
+        return services;
+    }
+    internal static IServiceCollection AddInMemoryDb(this IServiceCollection services) //TODO implement mock data
```

Comment:
You don't need this method in this project. And since you made it internal you cannot use it anywhere else. In Memory db should be used when running tests. But test are usually made in their own project.  https://docs.microsoft.com/en-us/visualstudio/test/walkthrough-creating-and-running-unit-tests-for-managed-code?view=vs-2022

Reply:
Yeah ok, I forgot about this, I remember, sorry.